client/my-sites/current-site/index.jsx

```diff
@@ -130,9 +85,13 @@ class CurrentSite extends Component {
 				) : (
 					<AllSites />
 				) }
-
-				<SiteNotice site={ selectedSite } />
+				<AsyncLoad
+					require="my-sites/current-site/notice"
+					placeholder={ null }
```

Comment:
tangent, but maybe placeholder should default to null?

Reply:
Agreed, that would be a better API for most cases. I'm thinking of `placeholder` prop behaving as follows:
- `<AsyncLoad />`: no `placeholder` prop, no presence in DOM while loading.
- `<AsyncLoad placeholder />`: `true` bool value of the prop, default placeholder.
- `<AsyncLoad placeholder={ <MyPlaceholder /> } />`: custom element as placeholder.

Migrating to `React.lazy` is also worth exploring.